package.json

```diff
@@ -93,6 +94,7 @@
     "grunt-jsonlint": "1.0.7",
     "grunt-newer": "1.2.0",
     "grunt-nsp": "2.2.0",
+    "grunt-remarkable": "1.0.0",
```

Comment:
Hhmm, **marked** and **grunt-marked** were both dependencies, not devDependencies. Wonder if these need to be moved up to dependencies as well (which probably means regenerating the shrinkwrap file).